src/worldcereal/openeo/preprocessing.py

```diff
@@ -269,10 +269,11 @@ def raw_datacube_DEM(
                 "https://stac.openeo.vito.be/collections/COPERNICUS30_DEM_SLOPE",
                 spatial_extent=spatial_extent,
                 bands=["Slope"],
-            )
-            .rename_labels(dimension="bands", target=["slope"])
-            .min_time()
+            ).rename_labels(dimension="bands", target=["slope"])
+            # .min_time()
```

Comment:
I'm actually not sure this should still be commented. It was a workaround for a metadata issue. I think it should be back in.